harness/determined/exec/harness.py

```diff
@@ -221,6 +221,11 @@ def main() -> None:
         f"New trial runner in (container {container_id}) on agent {agent_id}: {env.__dict__}."
     )
 
+
+    if env.experiment_config['checkpoint_storage']['type'] == 'azure':
+        from azure.storage.blob import BlobServiceClient
+        logging.getLogger("azure").setLevel(logging.ERROR)
+
```

Comment:
I'm reasonably confident that you could replace this whole clause with just `logging.getLogger("azure").setLevel(logging.ERROR)` without issue